packages/pigeon/platform_tests/test_plugin/ios/Classes/TestPlugin.swift

```diff
@@ -208,6 +212,20 @@ public class TestPlugin: NSObject, FlutterPlugin, HostIntegrationCoreApi {
     }
   }
 
+  // TODO: (tarrinneal) Update error handling tests to properly recieve and handle errors.
+  // See issue https://github.com/flutter/flutter/issues/118243
+  func callFlutterThrowError(completion: @escaping (Result<Any?, Error>) -> Void) {
+    flutterAPI.throwError() { 
+      completion(.success($0))
```

Comment:
Same (although at least `success` will fail the test, so if you want to use this pattern instead of TODOs everywhere that's fine).